src/Type/Row.purs

```diff
@@ -70,3 +74,17 @@ instance listToRowCons
   :: ( ListToRow tail tailRow
      , RowCons label ty tailRow row )
   => ListToRow (Cons label ty tail) row
+
+
+class Homogeneous (row :: # Type) fieldType
```

Comment:
Perhaps a value-level function `fieldType :: RProxy row -> Proxy fieldType` would be handy as well.

Reply:
Will do.